testsuite/MDAnalysisTests/analysis/test_hole2.py

```diff
@@ -480,7 +480,7 @@ def test_over_order_parameters(self, hole):
             assert key == rmsd
 
         idx = np.argsort(op)
-        arr = np.array(list(hole.profiles.values()))
+        arr = np.array(list(hole.profiles.values()), dtype=object)
```

Comment:
I didn't explain this change; this is because numpy was complaining of a VisibledeprecationWarning on running the test ( I assume we weren't picking this up in Travis because the ci helpers tend to pin the numpy version to something conservative).